README.rst

```diff
@@ -0,0 +1,183 @@
+
+Namespace
+  The first part of a Fully Qualified Collection Name, the namespace usually reflects a functional content category. Example: in ``cisco.ios.ios_config``, “Cisco” is the Namespace. Namespaces are reserved and distributed by Red Hat at Red Hat’s discretion. Many, but not all, namespaces will correspond with vendor names. May be different from the GitHub organization name.
+
+Collection name
+  The second part of a Fully Qualified Collection Name, the collection name further divides the functional characteristics of the collection content and denotes ownership.  For example, the cisco namespace might contain  cisco.ios, ``cisco.ios_community``, and ``cisco.ios_prc``, containing content for managing ios network devices maintained by Cisco.
+
+The community.general collection
+  A special collection managed by the Ansible Community Team containing all the modules and plugins which shipped in Ansible 2.9 that don't have their own dedicated Collection. A work in progress can be found in `community.general <https://github.com/ansible-collection-migration/community.general/>`_ repository.
+
+Repository
+  The location of the source code included in a collection. Contributors make suggestions, fix bugs, and add features through the repository. Collection owners can host repositories on GitHub, Gerrit, or any other  source code repository platform they choose.
+ 
+ 
+Documentation
+==============
+
+* `Using Ansible Collections <https://docs.ansible.com/ansible/latest/user_guide/collections_using.html>`_
+* `Using Developing Collections <https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html>`_
+ 
+Work needed
+===========
+
+Ansible 2.9 already contains basic support for Collections.
+
+The majority of the Ansible 2.10 release cycle is for:
+
+* Defining what the split of collections should be
+* Defining which modules go into these new modules
+* Defining ansible-base (ie which modules stay in ansible/ansible)
+* Updating test infrastruture
+* Testing the changes
+* Getting feedack from *you*
+  
+
+We will soon begin the migration of content out of ansible/ansible, into its new component collection repositories.
+
+**Warning:** Dates subject to change
+
+* On 2nd March  2020, we will freeze the devel branch using protected branches, and we will create the temp-2.10-devel branch from devel. This date marks the end of merging non-base plugin/module PRs into ansible/ansible.
```

Comment:
If possible, it would be nice to have a list of what are considered the 'base plugin/modules' and link to that list here.